pkg/golinters/gomodguard.go

```diff
@@ -21,75 +24,70 @@ func NewGomodguard() *goanalysis.Linter {
 		issues   []goanalysis.Issue
 		mu       = sync.Mutex{}
 		analyzer = &analysis.Analyzer{
-			Name: goanalysis.TheOnlyAnalyzerName,
-			Doc:  goanalysis.TheOnlyanalyzerDoc,
+			Name: gomodguardName,
+			Doc:  gomodguardDesc,
```

Comment:
Could revert this change?
The `TheOnlyAnalyzer` is a specific pattern.

Reply:
Yep.

Reply:
Both or just the name?

Reply:
you can keep `goanalysis.TheOnlyanalyzerDoc` only

Reply:
Reverted both.